src/renderer/components/delete-cluster-dialog/__tests__/delete-cluster-dialog.test.tsx

```diff
@@ -91,6 +92,8 @@ describe("<DeleteClusterDialog />", () => {
   beforeEach(async () => {
     const { mainDi, runSetups } = getDisForUnitTesting({ doGeneralOverrides: true });
 
+    mainDi.override(createContextHandlerInjectable, () => () => undefined);
```

Comment:
Should this be done in `getDiForUnitTesting` so that it is always run by default?

Reply:
Oh yes, that is why I would put it behind some sort of flag (like `doGeneralOverrides` is). I suggested moving it because I think that the vast majority of tests would want it overridden.